crates/orchestrator/src/cli/mod.rs

```diff
@@ -0,0 +1,581 @@
+            .required(true)
+            .multiple(false)
+    ),
+    group(
+        ArgGroup::new("settlement_layer")
+            .args(&["settle_on_ethereum", "settle_on_starknet"])
+            .required(true)
+            .multiple(false)
+    ),
+    group(
+        ArgGroup::new("storage")
+            .args(&["aws_s3"])
+            .required(true)
+            .multiple(false)
+    ),
+    group(
+      ArgGroup::new("queue")
+          .args(&["aws_sqs"])
+          .required(true)
+          .multiple(false)
+    ),
+    group(
+      ArgGroup::new("alert")
+          .args(&["aws_sns"])
+          .required(true)
+          .multiple(false)
+    ),
+    group(
+        ArgGroup::new("prover")
+            .args(&["sharp"])
+            .required(true)
+            .multiple(false)
+    ),
+    group(
+        ArgGroup::new("da_layer")
+            .args(&["da_on_ethereum"])
+            .required(true)
+            .multiple(false)
+    ),
+    group(
+        ArgGroup::new("cron")
+            .args(&["aws_event_bridge"])
+            .required(true)
+            .multiple(false)
+    ),
+)]
+pub struct RunCmd {
+    // Run mode
+    #[arg(long)]
+    pub run: bool,
+
+    #[arg(long)]
+    pub setup: bool,
+
+    // AWS Config
+    #[clap(flatten)]
+    pub aws_config_args: AWSConfigCliArgs,
+
+    // Settlement Layer
+    #[command(flatten)]
+    ethereum_args: settlement::ethereum::EthereumSettlementCliArgs,
+
+    #[command(flatten)]
+    starknet_args: settlement::starknet::StarknetSettlementCliArgs,
+
+    // Storage
+    #[clap(flatten)]
+    pub aws_s3_args: storage::aws_s3::AWSS3CliArgs,
+
+    // Queue
+    #[clap(flatten)]
+    pub aws_sqs_args: queue::aws_sqs::AWSSQSCliArgs,
+
+    // Server
+    #[clap(flatten)]
+    pub server_args: server::ServerCliArgs,
+
+    // Alert
+    #[clap(flatten)]
+    pub aws_sns_args: alert::aws_sns::AWSSNSCliArgs,
+
+    // Database
+    #[clap(flatten)]
+    pub mongodb_args: database::mongodb::MongoDBCliArgs,
+
+    // Data Availability Layer
+    #[clap(flatten)]
+    pub ethereum_da_args: da::ethereum::EthereumDaCliArgs,
+
+    // Prover
+    #[clap(flatten)]
+    pub sharp_args: prover::sharp::SharpCliArgs,
+
+    // Cron
+    #[clap(flatten)]
+    pub aws_event_bridge_args: AWSEventBridgeCliArgs,
+
+    // SNOS
+    #[clap(flatten)]
+    pub snos_args: snos::SNOSCliArgs,
+
+    #[arg(env = "MADARA_ORCHESTRATOR_MADARA_RPC_URL", long, required = true)]
+    pub madara_rpc_url: Url,
+
+    // Service
+    #[clap(flatten)]
+    pub service_args: service::ServiceCliArgs,
+    #[clap(flatten)]
+    pub instrumentation_args: instrumentation::InstrumentationCliArgs,
+}
+
+impl RunCmd {
+    pub fn validate_aws_config_params(&self) -> Result<AWSConfigParams, String> {
+        let aws_endpoint_url =
+            Url::parse("http://localhost.localstack.cloud:4566").expect("Failed to parse AWS endpoint URL");
+        let aws_default_region = "localhost".to_string();
```

Comment:
this will break in production

Reply:
aws_default_region should be set to aws_region

Reply:
Valid, moreover there's no point of sending them over cli, since they are automatically picked up by external libraries using env, so we'll have to add to env only
Hence removed from CLI completely